test/integration/emulator/TimeTurner.cpp

```diff
@@ -160,7 +165,7 @@ void TimeTurner::advance()
 
 void TimeTurner::advance(uint64_t nanoSeconds)
 {
-    nanoSeconds = std::max(nanoSeconds, 2 * utils::Time::ms);
+    nanoSeconds = std::max(nanoSeconds, _granularity);
```

Comment:
granularity allows timer source o advance in smaller steps. I noticed this caused physically impossible behaviour on low bw links in emulation.